packages/backend/src/utils/QueryBuilder/queryBuilder.test.ts

```diff
@@ -820,3 +822,151 @@ describe('Query builder', () => {
         });
     });
 });
+
+describe('Escaping in postgres', () => {
+    const postgresSqlBuilder = new PostgresSqlBuilder();
```

Comment:
initialize new PostgresSqlBuilder, rather than in the mock (see previous comment)